{
    "CVE_data_meta": {
        "ASSIGNER": "cve-assign@distributedweaknessfiling.org",
        "ID": "CVE-2019-1010142",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "SecDev",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "scapy",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "2.4.0"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "scapy 2.4.0 is affected by: Denial of Service. The impact is: infinite loop, resource consumption and program unresponsive. The component is: _RADIUSAttrPacketListField.getfield(self..). The attack vector is: over the network or in a pcap. both work."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "Denial of Service"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "url": "https://github.com/secdev/scapy/pull/1409/files#diff-441eff981e466959968111fc6314fe93L1058",
                "refsource": "MISC",
                "name": "https://github.com/secdev/scapy/pull/1409/files#diff-441eff981e466959968111fc6314fe93L1058"
            },
            {
                "refsource": "MISC",
                "name": "https://www.imperva.com/blog/scapy-sploit-python-network-tool-is-vulnerable-to-denial-of-service-dos-attack-cve-pending/",
                "url": "https://www.imperva.com/blog/scapy-sploit-python-network-tool-is-vulnerable-to-denial-of-service-dos-attack-cve-pending/"
            },
            {
                "refsource": "MISC",
                "name": "https://github.com/secdev/scapy/pull/1409",
                "url": "https://github.com/secdev/scapy/pull/1409"
            },
            {
                "refsource": "BID",
                "name": "106674",
                "url": "http://www.securityfocus.com/bid/106674"
            }
        ]
    }
}